src/app/sign-up-metamask/sign-up-metamask.component.html

```diff
@@ -50,7 +50,7 @@
 
       <div *ngIf="metamaskState === METAMASK.ERROR">
         <div class="text-danger">{{ errorMessage }}</div>
-        <div class="text-warning text-xs">{{ warningMessage }}</div>
+        <!-- <div class="text-warning text-xs">{{ warningMessage }}</div> -->
```

Comment:
What's the warning message?

Reply:
Was planning on including warnings along side error messages but decided to just use errors instead so this can be removed